config/media.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Media Config
+ */
+
+return array(
+
+	// Where to upload media files eg. images. Take note of the trailing slash.
+	// This should be in the Document root.
+	'media_upload_dir' => 'media/uploads',
+
+	// Maximum file upload size in bytes. Remember this figure should not be larger
+	// than the maximum file upload size set on the server. 1Mb by default.
+	'max_upload_bytes' => '1048576',
```

Comment:
Can we add a todo that setting bytes is gross and should be made more logical somehow? I imagine this is something people may want to readily configure, which means they have to either know base 10 to an even number off by heart or google it.